prioritize.py

```diff
@@ -26,12 +30,95 @@
 )
 
 
+# Priorities
+prioritiesT = List[float]
+
+# Normalized priorities (between [Ø, length of queue])
+priorities_normT = List[int]
+
+
+class Queue:
```

Comment:
Why is there a need to implement our own queue (there is one in Python: https://docs.python.org/fr/3/library/queue.html )? At least it may be inherited from it. If this does not fit our needs, at least we should explain in the docstring how it differs.

Reply:
The two essentials properties missing from queues in the standard libraries are implemented here in the methods `remove` and `to_priority_list`.

`remove` is needed when multiple queue are combined.

Reply:
So I would rename as `PriorityQueue` and put a word in the docstring that it adds a `remove` method that's not present in standard queues. You can also rename `to_priority_list` as `to_prioritized_list`